chips/nrf5x/src/timer.rs

```diff
@@ -293,6 +293,10 @@ impl<'a> TimerAlarm<'a> {
         });
     }
 
+    pub fn get_current_time(&self) -> hil::time::Ticks32 {
```

Comment:
Why is this useful if it's just a wrapper for `self.now()`? You can always get access to the `now` method with just `use kernel::hil::time::Time`